packit/cli/init_local_yaml.py

```diff
@@ -0,0 +1,65 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import logging
+from pathlib import Path
+
+import click
+
+from packit.config import (
+    get_context_settings,
+    pass_config,
+)
+
+logger = logging.getLogger(__name__)
+
+
+@click.command(
+    "init-local-config",
+    context_settings=get_context_settings(),
+    short_help="Creates a localyaml config file for and ensures it's not version controlled",
+)
+@pass_config
+def init_local_yaml(config):
+    """Initialize packit-local yaml configuration file."""
+    config_path = Path(".packitLocal.yaml")
+
+    if config_path.exists() and not click.confirm(
+        f"{config_path} already exists. Overwrite?",
+    ):
+        return
+
+    template = """
+        # Packit local configuration (not version controlled)
+        # This file overrides user and system configuration for this project only
+
+        # Authentication (optional - overrides ~/.config/packit.yaml)
+        # authentication:
+        #   github.com:
+        #     token: YOUR_TOKEN
```

Comment:
I don't think that this is the exact goal of this feature, and also I have a feeling that if you go in this direction, you might complicate this issue **by a lot** for yourself.

If I understood correctly, the aim was to provide a way to configure Packit **jobs** that can be run only locally.